flexget/components/jellyfin/api_jellyfin.py

```diff
@@ -0,0 +1,2149 @@
+from abc import ABC, abstractmethod, abstractstaticmethod
+from collections.abc import MutableSet
+from datetime import datetime
+import re
+from urllib.parse import urlencode
+
+import functools
+from requests.exceptions import RequestException, HTTPError
+from loguru import logger
+
+from flexget.utils.tools import get_current_flexget_version, split_title_year
+from flexget.utils import requests
+from flexget.plugin import PluginError
+from flexget.entry import Entry
+from flexget.utils.simple_persistence import SimplePersistence
+from flexget.components.jellyfin.jellyfin_util import get_field_map
+
+persist = SimplePersistence('api_jellyfin')
+
+JELLYFIN_ENDPOINT_LOGIN = '/emby/Users/AuthenticateByName'
```

Comment:
`/emby` should be removed. We still have legacy re-routing to /emby but it will not live forever.

Reply:
i will remove the /emby and check if it still works.

Reply:
removed legacy /emby from jellyfin plugin